setup.cfg

```diff
@@ -47,3 +47,4 @@ dev =
 lint =
     mypy
     pre-commit
+psqlgraph_test_utils =
```

Comment:
What does this line do?

Reply:
nothing, just a good practice to have it.

Reply:
https://github.com/NCI-GDC/psqlgraph/pull/227#discussion_r1265554045